python/tskit/drawing.py

```diff
@@ -178,14 +188,26 @@ def __init__(
         node_labels=None,
         mutation_labels=None,
         node_attrs=None,
+        mutation_attrs=None,
         edge_attrs=None,
         node_label_attrs=None,
+        mutation_label_attrs=None,
+        root_svg_attributes=None,
+        style=None,
     ):
         self.ts = ts
         if size is None:
             size = (200 * ts.num_trees, 200)
+        if root_svg_attributes is None:
+            root_svg_attributes = {}
+        if max_tree_height is None:
+            max_tree_height = "ts"
```

Comment:
Bitter experience has shown that putting in hard coded values as the default in complicated functions is a Bad Idea. The problem is that you have no way of knowing whether the user specifically asked for "ts" or just said "give me the default". Unless you are **certain** that this will always be the appropriate default, in the presence of all possible future arguments and their possible value assignments, using ``None`` as the default is the way to go.